addons/microsoft_calendar/models/calendar.py

```diff
@@ -54,10 +56,26 @@ def write(self, values):
             if previous_event_before_write != previous_event_after_write:
                 # Outlook returns a 400 error if you try to synchronize an occurrence of this type.
                 raise UserError(_("Modified occurrence is crossing or overlapping adjacent occurrence."))
-        return super().write(values)
+        notify_context = self.env.context.get('dont_notify', False)
+        res = super(Meeting, self.with_context(dont_notify=notify_context)).write(values)
+        if recurrence_update_setting in ('all_events',) and len(self) == 1 and values.keys() & self._get_microsoft_synced_fields():
```

Comment:
recurrence_update_setting in ('all_events',) can't be in all_events because updating the whole recurrence was introduced in 15.0.